docs/service-loadbalancer.md

```diff
@@ -0,0 +1,289 @@
+# Service of type LoadBalancer
+
+## Table of Contents
+
+<!-- toc -->
+- [Service external IP management by Antrea](#service-external-ip-management-by-antrea)
+  - [Configuration](#configuration)
+    - [Enable Service external IP management feature of `antrea-agent`](#enable-service-external-ip-management-feature-of-antrea-agent)
+	- [Create an ExternalIPPool custom resource](#create-an-externalippool-custom-resource)
+	- [Create a Service of type LoadBalancer](#create-a-service-of-type-loadbalancer)
+	- [Validate Service external IP](#validate-service-external-ip)
+  - [Limitations](#limitations)
+- [Using MetalLB with Antrea](#using-metallb-with-antrea)
```

Comment:
We missed two subsections under this section.

Reply:
fixed. thanks!